setup.py

```diff
@@ -96,7 +97,9 @@ def run(self):
             libp2p_build_install()
         else:
             libp2p_download_install()
-        proto_compile(os.path.join(self.build_lib, 'hivemind', 'proto'))
+
+        path = self.build_lib if os.path.exists(self.build_lib) else here
+        proto_compile(os.path.join(path, 'hivemind', 'proto'))
```

Comment:
Have you checked that this works when installing both in editable and non-editable mode? (preferably, install hivemind without `-e` from source and then make sure that `_pb2.py` files are located in site-packages)

Reply:
Yes, we did